pandas/core/series.py

```diff
@@ -47,34 +56,25 @@
                                 _maybe_box_datetimelike,
                                 _dict_compat,
                                 standardize_mapping)
-from pandas.core.index import (Index, MultiIndex, InvalidIndexError,
-                               Float64Index, _ensure_index)
-from pandas.core.indexing import check_bool_indexer, maybe_convert_indices
-from pandas.core import generic, base
+from pandas.core.config import get_option
 from pandas.core.internals import SingleBlockManager
 from pandas.core.categorical import Categorical, CategoricalAccessor
-import pandas.core.strings as strings
-from pandas.core.indexes.accessors import (
-    maybe_to_datetimelike, CombinedDatetimelikeProperties)
+
```

Comment:
ideally don't move imports around unless they are germane, it makes the diff hard to read